server/models/User.ts

```diff
@@ -0,0 +1,13 @@
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  handle: {type: String, required: true, unique: true},
```

Comment:
Let's remove things we don't need from the schema (handle, bio, listings). 

See issue #3 for the query information that needs to be tracked in the database for now.